src/components/v2/V2Project/V2PayButton/V2PayForm/StickerSelection.tsx

```diff
@@ -0,0 +1,46 @@
+import { Space } from 'antd'
+import { CloseCircleFilled } from '@ant-design/icons'
+import { IconedImage } from 'components/IconedImage'
+import { ThemeContext } from 'contexts/themeContext'
+import { useContext } from 'react'
+
+export const StickerSelection = ({
+  value,
+  onChange,
+}: {
+  value?: string[]
+  onChange?: (value: string[]) => void
+}) => {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  const handleImageDeletion = (i: number) => {
+    if (!value) {
+      onChange?.([])
+      return
+    }
+    const editedImages = [...value.slice(0, i), ...value.slice(i + 1)]
+    onChange?.(editedImages)
+  }
+  return (
+    <>
```

Comment:
**suggestion**: think we can remove the fragment here.

Reply:
we can! :)